binders/kafka-binder/spring-cloud-stream-binder-kafka-reactive/src/main/java/org/springframework/cloud/stream/binder/reactorkafka/ReactorKafkaBinderConfiguration.java

```diff
@@ -28,12 +27,16 @@
 import org.springframework.cloud.stream.binder.kafka.provisioning.KafkaTopicProvisioner;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Import;
 
+/**
+ * Binder configuration for ReactorKafka.
+ *
+ * @author Gary Russell
+ * @author Chris Bono
+ */
 @Configuration(proxyBeanMethods = false)
 @ConditionalOnMissingBean(Binder.class)
-@Import({ KafkaAutoConfiguration.class })
-@EnableConfigurationProperties({ KafkaExtendedBindingProperties.class })
+@EnableConfigurationProperties({ KafkaProperties.class, KafkaExtendedBindingProperties.class })
```

Comment:
Same approach that you will see throughout the binder configs "Replace the KAC w/ KP".